staging/src/k8s.io/client-go/tools/portforward/portforward.go

```diff
@@ -197,8 +197,9 @@ func (pf *PortForwarder) forward() error {
 	var err error
 
 	listenSuccess := false
-	for _, port := range pf.ports {
-		err = pf.listenOnPort(&port)
+	for i, _ := range pf.ports {
+		port := &pf.ports[i]
```

Comment:
Ouch - nice find.  We really need to see if there's a static analyzer that can catch this!